app/src/utils/address.ts

```diff
@@ -91,3 +91,13 @@ export const getRecipientAddress = (manualRecipient: ManualRecipient, wallet?: W
 /** Get the transfer sender address from the sender origin base (Substrate or Ethereum)*/
 export const getSenderAddress = async (sender: Sender): Promise<string> =>
   sender instanceof JsonRpcSigner ? await sender.getAddress() : (sender as InjectedAccount).address
+
+/** Get an example address for a specific type. Can be used to prefetch the fees before address input, as fees shouldn't differ. */
+export const getExampleAddress = (type: AddressType): string => {
```

Comment:
The right word here is _placeholder_ 

```suggestion
/** Get a placeholder address for a specific type. Can be used to prefetch the fees before address input, as fees shouldn't differ. */
export const getPlaceholderAddress = (type: AddressType): string => {
```